app/views/mfa/confirm.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= form_tag("/mfa/confirm/#{@user.id}", :class => 'col-md-3 col-md-offset-4') do %>
```

Comment:
Instead of using this page just redirect to the main dashboard and use the 'message' function to show that an mfa has been successfully created